src/Okta.Sdk.IntegrationTests/FeaturesScenarios.cs

```diff
@@ -0,0 +1,61 @@
+﻿// <copyright file="FeaturesScenarios.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class FeaturesScenarios
+    {
+        [Fact]
+        public async Task ListFeatures()
+        {
+            var client = TestClient.Create();
+            var features = await client.Features.ListFeatures().ToListAsync();
```

Comment:
I think you have to create at least a feature here before listing, otherwise, we are trusting in something we don't have control.

Reply:
Generally I would agree; however in my own experimentation/testing, it appears that features themselves are integral to the system, without which we wouldn't have sets of valid offerings.  Also, I try to avoid making data updates that may be difficult to clean up later.